kong/tools/public.lua

```diff
@@ -53,6 +53,11 @@ do
 
     [MIME_TYPES.json] = function()
       local raw_body  = ngx_req_get_body_data()
+      if not raw_body then
```

Comment:
We need the same check and error log up in L.48 where we call `get_body_data()` on multipart payloads. Otherwise, lua-multipart will fail down in its execution path (even if it were sanitizing its input, it would have to throw an error upon an invalid `body` argument.

Reply:
ok